src/protect-function-app/README.md

```diff
@@ -0,0 +1,124 @@
+<!-- TO PUBLISH TO DOCS SAMPLES BROWSER: UPDATE AND THEN UNCOMMENT THE YAML FRONT-MATTER SECTION BELOW -->
+<!--
+# Metadata required by https://docs.microsoft.com/samples/browse/
+# Metadata properties: https://review.docs.microsoft.com/help/contribute/samples/process/onboarding?branch=main#add-metadata-to-readme
+languages:
+- csharp
+page_type: sample
+name: "C# Azure Function that protects an HTTP trigger function with Easy Auth and access token scope validation."
+description: "This C# Azure Function protects its own HTTP Trigger function with Easy Auth and access token scope validation. The code in this sample is used by one or more articles on docs.microsoft.com."
+products:
+- azure
+- azure-active-directory
+- azure-functions
+urlFragment: ms-identity-docs-code-csharp
+---
+-->
+# C# | Azure Function | protect an API | Microsoft identity platform
+
+<!-- Build badges here
+![Build passing.](https://img.shields.io/badge/build-passing-brightgreen.svg) ![Code coverage.](https://img.shields.io/badge/coverage-100%25-brightgreen.svg) ![License.](https://img.shields.io/badge/license-MIT-green.svg)
+-->
+
+This C# Azure Function protects its own API (HTTP trigger) with the combination of the built-in authentication & authorization feature of Azure Functions (commonly known as "Easy Auth") and JWT access token scope validation.
+
+```console
+$ curl https://<your-function>.azurewebsites.net/api/greeting -H "Authorization: Bearer <valid-access-token>"
+Hello, world. You were able to access this because you provided a valid access token with the Greeting.Read scope as a claim.
+```
+
+> :page_with_curl: This sample application backs one or more technical articles on docs.microsoft.com. <!-- TODO: Link to first tutorial in series when published. -->
+
+## Prerequisites
+
+- Azure Active Directory (Azure AD) tenant and the permissions or role required for managing app registrations in the tenant.
+- [Download .NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+- [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local)
+- An empty, [C# Function app (v4) deployed to Azure](https://docs.microsoft.com/azure/azure-functions/create-first-function-cli-csharp) and the permissions or role required to modify its settings
```

Comment:
Looks like v4 is called out here, which is great.  (Nothing to change, just an observation)